src/expression.d

```diff
@@ -2893,6 +2893,11 @@ extern (C++) abstract class Expression : RootObject
     final void checkDeprecated(Scope* sc, Dsymbol s)
     {
         s.checkDeprecated(loc, sc);
+        // Checks for deprecated and disabled have historically
+        // been in the same function, this is a temporary transition
+        // measure to keep the same semantic.
```

Comment:
It bothers me that checking for **disabled** is inside a function called checking for **deprecated**. Those two are quite different things. I don't think combining them improves coherence of the code.